src/CommunityToolkit.Maui.Markup/TypedBinding.cs

```diff
@@ -248,7 +248,7 @@ void ApplyCore(TSource? sourceObject, BindableObject target, BindableProperty pr
 		var needsSetter = (mode == BindingMode.TwoWay && fromTarget) || mode == BindingMode.OneWayToSource;
 		if (needsSetter && sourceObject is not null)
 		{
-			var value = GetTargetValue(target.GetValue(property), typeof(TProperty)) ?? throw new InvalidOperationException("Unable to find target value");
+			var value = GetTargetValue(target.GetValue(property), typeof(TProperty));
```

Comment:
I wonder if we should check whether `TProperty` is nullable in this scenario. Thoughts @TheCodeTraveler?